### DISCUSSION TITLE
Duplicate env-s button

### DISCUSSION THREAD

**Discussion by newadbie** [Infisical/infisical#487] [Feature Request]
Hey folks, is there any possibilities that you will introduce in future "duplicate" button? I think it would by very hand if i would be able to copy all env-s just with one click, for example i am implementing a very big feature but i am working just with my local env and i don't want to push stuff to my company mates env-s, i just would like to create duplicate and work with it

**Comment by vmatsiiako** [comment]
Hey @newadbie! Yes, this is coming very soon (within the next few days)! :)
